certified-connectors/Impower/apiDefinition.swagger.json

```diff
@@ -1,3188 +1,3682 @@
+      "post": {
+        "tags": [
+          "Connection"
+        ],
+        "summary": "Create an application connection",
+        "operationId": "createConnectionUsingPOST",
+        "x-ms-trigger": "single",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "connectionCreateDto",
+            "description": "Connection data.\nid | name | description\n---|---|---\n4 | Powerautomate | Microsoft connected applications\n8 | REST App | Use 8 for generic REST applications",
+            "x-ms-summary": "Connection data.\nid | name | description\n---|---|---\n4 | Powerautomate | Microsoft connected applications\n8 | REST App | Use 8 for generic REST applications",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConnectionCreateDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "x-ms-summary": "OK",
+            "schema": {
+              "$ref": "#/definitions/ConnectionDto"
             }
+          }
         },
-        "/v2/contacts/{contactId}": {
-            "get": {
-                "tags": [
-                    "Contact API"
-                ],
-                "summary": "Get a contact by the given id",
-                "description": "Retrieves a contact by the given id",
-                "x-ms-summary": "Retrieves a contact by the given id",
-                "operationId": "getContactByIdUsingGET",
-                "produces": [
-                    "application/json"
-                ],
-                "parameters": [
-                    {
-                        "name": "contactId",
-                        "in": "path",
-                        "x-ms-url-encoding": "single",
-                        "description": "Unique identifier of a contact",
-                        "x-ms-summary": "Unique identifier of a contact",
-                        "required": true,
-                        "type": "integer",
-                        "format": "int32"
-                    }
-                ],
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "x-ms-summary": "OK",
-                        "schema": {
-                            "$ref": "#/definitions/ContactDto"
-                        }
-                    },
-                    "404": {
-                        "description": "Not Found",
-                        "x-ms-summary": "Not Found"
-                    }
-                },
-                "deprecated": false
+        "security": [
+          {
+            "API Key": []
+          }
+        ],
+        "deprecated": false
+      }
+    },
+    "/v2/connections/{connectionIdX}": {
+      "put": {
```

Comment:
Can you please add description property? 

Please refer the schema and template for the help.

Error - ##[error]The 'description' property is required. ValuePath: /paths/~1v2~1connections~1{connectionIdX}/put

Schema - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json

Template - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

Reply:
added description property